pom.xml

```diff
@@ -643,10 +674,42 @@
     </dependencies>
 
     <repositories>
+        <repository>
+            <id>artifacts.oicr.on.ca</id>
+            <name>artifacts.oicr.on.ca</name>
+            <url>https://artifacts.oicr.on.ca/artifactory/collab-release</url>
+        </repository>
+        <repository>
+            <id>seqwaremaven.oicr.on.ca</id>
+            <name>seqwaremaven.oicr.on.ca-releases</name>
+            <url>https://seqwaremaven.oicr.on.ca/artifactory/seqware-dependencies</url>
+        </repository>
+        <repository>
+            <id>eclipse-egit-github</id>
+            <name>Eclipse egit github repository</name>
+            <url>https://repo.eclipse.org/content/repositories/egit-releases</url>
+        </repository>
+        <!-- broad -->
+        <repository>
+            <id>artifactory.broadinstitute.org</id>
+            <name>artifactory.broadinstitute.org</name>
+            <url>https://artifactory.broadinstitute.org/artifactory/libs-release</url>
+            <releases>
+                <enabled>true</enabled>
+            </releases>
+            <snapshots>
+                <enabled>false</enabled>
+            </snapshots>
+        </repository>
+        <repository>
+            <id>sagebionetworks</id>
+            <name>sagebionetworks-releases</name>
+            <url>http://sagebionetworks.artifactoryonline.com/sagebionetworks/libs-releases-local</url>
+        </repository>
         <repository>
             <id>seqware.sourceforge.net</id>
             <name>SeqWare Maven Repository</name>
-            <url>http://seqwaremaven.oicr.on.ca/artifactory/seqware-release</url>
```

Comment:
Re: travis conversation 

This breaks things because youxia is hosted on seqware-release, not collab-release. 
If you need things from collab-release, add that as a new repo rather than changing the existing one

Reply:
Thanks Denis, Took that approach. And added a new repo.

Reply:
:+1: